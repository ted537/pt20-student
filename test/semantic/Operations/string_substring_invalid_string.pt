using input;
var subStr = 5 / 2 : 5;
