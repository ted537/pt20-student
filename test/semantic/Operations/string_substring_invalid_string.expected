   .tFileDescriptor
   .tLiteralInteger
   oEmitValue
   % value emitted 1
   .tFileBind
   .tLiteralAddress
   oEmitDataAddress
   % value emitted 0
   .tStoreInteger
   .tInitialValue
     .tLiteralInteger
     oEmitValue
     % value emitted 5
     .tLiteralInteger
     oEmitValue
     % value emitted 2
     .tLiteralInteger
     oEmitValue
     % value emitted 5
     .tSubstring
     #eTypeMismatch
   .tInitEnd
   .tLiteralAddress
    .tStoreChar
 .tTrapBegin
 .tTrap
 oEmitTrapKind(trHalt)
 % value emitted 0
